tests/integration/kubernetes/confidential/k8s-confidential.bats

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+IMAGE_REPO="ghcr.io/confidential-containers/test-container"
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	( [ "${KATA_HYPERVISOR}" == "firecracker" ] || [ "${KATA_HYPERVISOR}" == "dragonball" ] || [ "${KATA_HYPERVISOR}" == "qemu-tdx" ] || [ "${KATA_HYPERVISOR}" == "qemu-snp" ] ) && skip "Test not supported yet."
+
+	pod_name="confidential-unencrypted-test"
+	container_name="confidential-unencrypted-container"
+	get_pod_config_dir
+
+	k8s_generate_service_yaml "${TEST_DIR}/${pod_name}.yaml" "${IMAGE_REPO}:unencrypted"
+	k8s_yaml_set_annotation "${TEST_DIR}/${pod_name}.yaml" "io.katacontainers.config.pre_attestation.enabled" "false"
+}
+
+@test "Test unencrypted confidential container launch success." {
```

Comment:
You skipped the setup, but the test would still run with the setup skipped.

Reply:
Fixed.